packages/suite/src/views/wallet/send/Outputs/Amount/TokenSelect.tsx

```diff
@@ -148,11 +149,15 @@ export const TokenSelect = ({ output, outputId }: TokenSelectProps) => {
     }, [outputId, tokenWatch, setAmount, getValues, account.networkType, isSetMaxActive]);
 
     useEffect(() => {
-        if (routerParams?.contractAddress) {
-            setValue(tokenInputName, routerParams.contractAddress, { shouldValidate: true });
+        if (sendFormPrefill) {
+            setValue(tokenInputName, sendFormPrefill, { shouldValidate: true });
```

Comment:
I am missing `Clear all` button in send form when token is autofilled
![Screenshot 2024-08-19 at 17 35 40](https://github.com/user-attachments/assets/614ce6d2-5d37-4616-8d42-0743b39c9016)

![Screenshot 2024-08-19 at 17 31 19](https://github.com/user-attachments/assets/a87a9550-2b5f-4634-b927-2c2187d2492a)

Reply:
Fixed